IceCubesApp/App/Tabs/Settings/ContentSettingsView.swift

```diff
@@ -20,6 +20,9 @@ struct ContentSettingsView: View {
       }.listRowBackground(theme.primaryBackgroundColor)
 
       Section("settings.content.media") {
+        Toggle(isOn: $userPreferences.autoPlayVideo) {
+          Text("settings.other.autoplay-video")
```

Comment:
I think this should then change the localization key to `setting.content.media.autoplay-video`

Reply:
Make it harder to keep track for when I'll merge adding other localization, like the ongoing Ukrainian one. I'm in favor of not moving the key if the value didn't change or/and still mean the same thing.

Reply:
Valid point. But the localizations need some clean-up at some time I think, I noticed, that in some languages they are grouped differently or have empty lines for grouping whereas in other languages this is not the case.

Reply:
I do agree, and it would be good to have a common (free) tools to do that but I'm not aware of one :(